### PR TITLE
Create a golem::add_file function

### LINKED ISSUE #837: golem::add_file
## Feature 

I want to be able to upload any file inside www, not juste css or JS

## TODO

+ [ ] Will behave as other writing functions, except it doesn't check for the extension

### PR DESCRIPTION
**Pull request by ArthurData**
close #837

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -111,6 +113,7 @@ importFrom(shiny,addResourcePath)
 importFrom(shiny,getShinyOption)
 importFrom(shiny,includeScript)
 importFrom(shiny,tags)
+importFrom(tools,file_ext)
```
Comment (ColinFay) [inline NAMESPACE:116]: Can you use the internal file_ext() function(in utils.R) instead ? This allows to remove tools from deps

Comment (ArthurData) [reply NAMESPACE]: I remove the importFrom from both `NAMESPACE` and `add_files.R`

**Comment by ColinFay** [comment]
@ArthurData I'm finishing this and merging, thanks for your help :)

**Comment by ColinFay** [comment]
Cherry picked in https://github.com/ThinkR-open/golem/tree/938-cherry-picked

**Pull request closed without merging**